### ISSUE TITLE
Allow totally clean font size list

### ISSUE THREAD

**Issue by dakom** [jewish-interactive/ji-wysiwyg-editor#17]
Similar to #3 but for font sizes

Should allow any string (i.e. if this option is used, it's up to the caller to decide `px`,`ems`, etc.)

**Comment by jpuri** [comment]
@dakom : whats possible with the current font-size option is that we change the available font sizes or reduce/add more to them. It does not have option to set `px,ems`. It would be required to develop a custom component for this - which will take longer.

**Comment by dakom** [comment]
OK - understood. Do you expect the list as numbers then, as opposed to strings?

**Comment by jpuri** [comment]
Yes currently its a dropdown where user selects a number and `px` is always added to it while styling.
